src/index.ts

```diff
@@ -21,6 +22,89 @@ function writeLoaders(args: CLIArgs) {
     fs.writeFileSync(args.output, output);
 }
 
+/**
+ * If requiredList only contains responseKey
+ */
+function validRequiredList(requiredList: Array<string>, responseKey: string) {
+    if (requiredList.length > 1) return false;
+    if (requiredList.length == 1 && requiredList[0] !== responseKey) {
+        return false;
+    }
+    return true;
+}
+
+/**
+ * Find all values of the intput key in an object recursively
```

Comment:
```suggestion
 * Find all values of the input key in an object recursively
```

would also benefit from an example, i'm not sure what this code is supposed to do by looking at it